Does not compile

Hello,
I've just cloned your repo and I got : 

```
$ ./gradlew clean build
Download https://repo1.maven.org/maven2/org/apache/tomcat/tomcat-annotations-api/8.5.28/tomcat-annotations-api-8.5.28.jar
e: /Users/adah/workspaces/spring-gradle-kotlin-multimodule/item/src/main/kotlin/cz/zubal/spring/multimodule/item/RepositoryItemService.kt: (3, 36): Unresolved reference: customer
e: /Users/adah/workspaces/spring-gradle-kotlin-multimodule/item/src/main/kotlin/cz/zubal/spring/multimodule/item/RepositoryItemService.kt: (20, 60): Unresolved reference: Customer

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':item:compileKotlin'.
> Compilation error. See log for more details

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED in 14s
6 actionable tasks: 3 executed, 3 up-to-date

```

--- a/build.gradle
+++ b/build.gradle
@@ -17,7 +17,6 @@ allprojects {
   apply plugin: 'kotlin'
   apply plugin: 'kotlin-spring'
   apply plugin: 'idea'
-  apply plugin: 'org.springframework.boot'
   apply plugin: 'io.spring.dependency-management'
 
   group = 'cz.zubal'
@@ -40,6 +39,9 @@ allprojects {
     mavenCentral()
   }
 
+  dependencyManagement {
+    imports { mavenBom("org.springframework.boot:spring-boot-dependencies:${springBootVersion}") }
+  }
 
   dependencies {
     compile('org.springframework.boot:spring-boot-starter-data-jpa')

